src/rows.jl

```diff
@@ -227,6 +227,28 @@ function Base.show(io::IO, row::Row)
     return nothing
 end
 
+"""
+    schema_field_names(::Type{<:Legolas.Schema})
+
+Get a tuple with the names of the fields of this `Legolas.Schema`, including names that
+have been inherited from this `Legolas.Schema`'s parent schema.
+"""
+schema_field_names(::Type{S}) where {S<:Schema} = throw(UnknownSchemaError(S()))
```

Comment:
I actually think the `MethodError` you'd get from removing this fallback method is more correct than an `UnknownSchemaError`; it could be that a schema exists (and thus is "known") but a method for `schema_field_names` simply wasn't defined for it.

Reply:
> it could be that a schema exists (and thus is "known") but a method for schema_field_names simply wasn't defined for it.

Hmmm how so?

I maybe skimmed too fast but using `UnknownSchemaError` here seems to be consistent with the other usages of it (and is useful here for the same reason it's useful in those other contexts)

Reply:
> Hmmm how so?

Very hypothetical case of defining a schema without using `@row`.

If it's consistent and useful then that's good enough for me.